src/jekyll/zeppelin.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+displayTitle: Apache Zeppelin running with Spark on Kubernetes
+title: Apache Zeppelin running with Spark on Kubernetes
+description: User Documentation for Apache Zeppelin running with Spark on Kubernetes
+---
+
+This page a ongoing effort to describe how to run Apache Zeppelin with Spark on Kubernetes
+
+> At the time being, the needed code is not integrated in the `master` branches of `apache-zeppelin` nor the `apache-spark-on-k8s/spark` repositories.
+> You are welcome to already ty it out and send any feedback and question.
+
+Firs things firs, you have to choose the following modes in which you will run Zeppelin with Spark on Kubernetes:
+
++ The `Kubernetes modes`: Can be `in-cluster` (within a Pod) or `out-cluster` (from outside the Kubernetes cluster).
```

Comment:
what are the proper terminology in k8s world? is "out-cluster" the right term?

Reply:
I had the same question and from the already used/seen `in-cluster`, I have deduced 'out-cluster`. Happy to change to any other more official terminology.

Reply:
@foxish @erikerlandson ?